docs/rose-suite-run-proposal/future-cli-conventions.md

```diff
@@ -0,0 +1,42 @@
+Placeholder - populate in new PR
```

Comment:
It is not anticipated that this is the finished form of the document: One of the tasks on the work plan is, after all, to summarize the debate in this document in another PR.